tests/roles/glance_adoption/tasks/main.yaml

```diff
@@ -9,11 +9,19 @@
         apiOverride:
           route: {}
         template:
+          customServiceConfig: |
```

Comment:
yeah, because the webhook will block you in advance. 
There's a plan to remove that default from `00-config.conf` because you always need an override anyway (it's just in the backlog)

Reply:
Ok but here we explicitly set "single" and webhook seems only block for "split" + file backend + rep > 0

Reply:
yeah, single would be sufficient here. I though that explicitly setting the config would be a nice thing to have for maintenance: for people not familiar w/ the glance-operator it's easy to understand the current CR config in the ansible playbook, but I agree we can try a variation of this procedure where we do not set `customServiceConfig`.

Reply:
so @karelyatin basically yeah, `type: single` should be sufficient

Reply:
Thanks, should be fine to keep it explicitly set.